doc/nvim-tree-lua.txt

```diff
@@ -494,6 +494,7 @@ applying configuration.
         },
         open_file = {
           quit_on_open = false,
+          prevent_buffer_override = true,
```

Comment:
buffer_override is used internally, however it's not a very clear description.

Perhaps we could call this `eject_buffer`?

Reply:
Yes, that's a better name. Maybe also `eject_tree` to highlight the nvim-tree buffer is ejected?

Reply:
false indicates "do nothing" i.e. don't interfere with normal vim behaviour. We never really do anything with the tree itself.

Since this is under `open_file` we could just call it `open_file.eject`. Your description is quite accurate, describing what happens to the new buffer.

I do get quite critical when it comes to option names / places. It's easy to get right when adding but very hard to refactor later...

Reply:
I agree naming things that you cannot change later should be discussed vigorously :) I will rename to `open_file.eject`